packages/desktop-client/src/components/budget/misc.js

```diff
@@ -513,13 +547,19 @@ export function SidebarGroup({
                       onEdit(group.id);
                     } else if (type === 'add-category') {
                       onShowNewCategory(group.id);
-                    } else {
+                    } else if (type === 'delete') {
                       onDelete(group.id);
+                    } else if (type === 'toggleHidden') {
+                      onSave({ ...group, hidden: !group.hidden });
                     }
                     setMenuOpen(false);
                   }}
                   items={[
                     { name: 'add-category', text: 'Add category' },
+                    {
+                      name: 'toggleHidden',
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
                      name: 'toggleVisibility',
```